INFO     D:\PyCharm\projects\API_yaml/logs/all.log:test_case.py:36 用例描述====>参数错误-失败
INFO     D:\PyCharm\projects\API_yaml/logs/all.log:testBase.py:120 请求方式====>GET
INFO     D:\PyCharm\projects\API_yaml/logs/all.log:testBase.py:121 请求地址====>http://api.21szxy.com/api/user/login?
INFO     D:\PyCharm\projects\API_yaml/logs/all.log:testBase.py:124 请求参数====>{
    "access_token": "Bj/7Asbw3x5dCpvTFTo9GWLduBL6KyimS2q7vY2I+FbYJgre1dDZ9C/i5h6ZvEZW",
    "account": 110256880,
    "password": "a123456",
    "baseid": 0,
    "schoolid": 461361
}
INFO     D:\PyCharm\projects\API_yaml/logs/all.log:testBase.py:131 响应状态码为====>200
INFO     D:\PyCharm\projects\API_yaml/logs/all.log:testBase.py:132 接口响应头为====>{'Server': 'nginx', 'Date': 'Fri, 22 Jan 2021 07:53:35 GMT', 'Content-Type': 'application/json;charset=UTF-8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'X-Application-Context': 'ms-bd:8084', 'Cache-Control': 'no-store', 'Content-Language': 'zh-CN'}
INFO     D:\PyCharm\projects\API_yaml/logs/all.log:testBase.py:133 接口响应内容====>{"message":"账号或密码错误","code":-1}
INFO     D:\PyCharm\projects\API_yaml/logs/all.log:testBase.py:86 断言： 预期值：'账号或密码错误' 实际值：'账号或密码错误', 断言成功!